### ISSUE TITLE
Unable to change fonts

### ISSUE THREAD

**Issue by stefanbreet** [kakawait/hugo-tranquilpeak-theme#259]
Thanks for making this theme available! I want to change the fonts from Open Sans and Merriweather to Lato and Roboto, but I'm unable to do so.

I tried to do so by editing the `_variables.scss` and `_fonts.scss` in two different ways: (1) by editing them directly in the theme directory, and (2) by adding them as custom css and referring to them in config.toml. Unfortunately, both approaches did not work.

What is the recommended procedure to change the fonts, font color, etc?

Thanks in advance for your help!

### Configuration
 - **Operating system with its version**: MacOS High Sierra 10.13.3
 - **Browser with its version**:  Chrome Version 64.0.3282.140 (Official Build) (64-bit)
 - **Hugo version**: 0.36
 - **Tranquilpeak version**: 0.4.3
 - **Do you reproduce on https://tranquilpeak.kakawait.com demo?**:  Not relevant

**Comment by newmanrs** [comment]
Was working on something like this today [with no real experience doing any web development or the node/npm ecosystems].  In the template directory you'll need to invoke the build script defined by gruntfile.js.  There are some details on how to do this given in the [dev docs](https://github.com/kakawait/hugo-tranquilpeak-theme/blob/31c71da9f5b37972ea649d7ae1b54c82e0d353e4/docs/developer.md#requirements). 

The gist is you'll to install `grunt-cli`, and then do something like a `npm install` in the root of this project folder and to regenerate the css use something like `grunt buildProd` (but at this stage read the dev docs).  

If you see weird errors check `node --version` and at least for me it seems to work great on `v8.9.4` but not on the most recent v9.?.? that I previously had installed.  Don't go cutting edge.

As to the edits you'll need to do in the scss files it's just importing the different google font (in `_fonts.scss`, and then changing up a bit of the variables file.

Best of luck, although this is a few weeks late I suppose.

**Comment by kakawait** [comment]
You can't reference `.scss` files directly. They must be compiled.

You can follow @newmanrs instruction in addition to existing dev docs.

@newmanrs I do not try yet with node v9. Maybe there is some issues

**Comment by stefanbreet** [comment]
Thanks for the clarification! I managed to figure it out, rebuilding it with grunt did the trick.

**Comment by fantasyao** [comment]
First of all, thank you for your discussion, but after reading this discussion, I still don't understand how to modify fonts,
My understanding after reading is:
1. It is necessary to modify the content in ('fonts. scss), as well as some variable files (what are these variable files specifically?)
2. Then use the grunt tool to execute (grunt buildProd) to rebuild the CSS of the theme,
3. Then restart 'hugo server-D'. And then it succeeded. Is my understanding correct?
I have no understanding of node and grunt at all. Grunt requires installing the node first and then executing 'npm install grunt cli g'. Is my understanding correct?
I have also reviewed the dev document and I am sorry, but I still don't know what I need to do after reading it
This is a portion of my 【 font. scss 】 content. I have already installed LXGW WenKai Mono.ttf on my computer.
```
@Font face{
Font family: 'LXGW WenKai Mono';
Font style: normal;
Font weight: 400;
Src: local ('LXGW WenKai Mono '), local ('LXGW WenKai Mono'), URL（ https://192960944.r.cdn36.com/chinesefonts3/packages/lxgwwenkai/dist/LXGWWenKai-Light/result.css ）Format ('truetype ');
}
```
At last, thank you all for your help again！
